Session/Media Viewing & Editing/MediaGalleryViewModel.swift

```diff
@@ -243,15 +252,27 @@ public class MediaGalleryViewModel {
             """
         }()
         
-        fileprivate static func filterSQL(threadId: String) -> SQL {
+        fileprivate static func filterSQL(threadId: String, mediaType: MediaType) -> SQL {
             let interaction: TypedTableAlias<Interaction> = TypedTableAlias()
             let attachment: TypedTableAlias<Attachment> = TypedTableAlias()
             
-            return SQL("""
-                \(attachment[.isVisualMedia]) = true AND
-                \(attachment[.isValid]) = true AND
-                \(interaction[.threadId]) = \(threadId)
-            """)
+            switch (mediaType) {
+                case .media:
+                    return SQL("""
+                        \(attachment[.isVisualMedia]) = true AND
+                        \(attachment[.isValid]) = true AND
+                        \(interaction[.threadId]) = \(threadId)
+                    """)
+                case .document:
+                    // FIXME: Remove "\(attachment[.sourceFilename]) <> 'session-audio-message'" when all platforms send the voice message properly
+                    return SQL("""
+                        \(attachment[.isVisualMedia]) = false AND
+                        \(attachment[.isValid]) = true AND
+                        \(attachment[.variant]) = \(Attachment.Variant.standard) AND
+                        \(attachment[.sourceFilename]) <> 'session-audio-message' AND
+                        \(interaction[.threadId]) = \(threadId)
```

Comment:
Please shift the `threadId` filter line above the `variant` and `sourceFilename` lines (we want to try filter indexed columns first before filtering on unindexed ones to minimise the performance impact)